packages/components/src/styles/components/stepper/index.scss

```diff
@@ -3,5 +3,5 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-@use "./step-indicator.scss";
-@use "./task-indicator.scss";
+@use "./step-indicator";
```

Comment:
I could undo these changes and make in a separate branch if that's more appropriate. (Removed `.scss` extensions to satisfy linter)

Reply:
I believe it could help to extract these (potentially with cherry-pick), so we can remove any unnecessary 'noise' from this PR. Let me know when it's up and I'll approve right-away. No need to undo them, as the changes should go away when you rebase the branch to main.

Reply:
Merged in then rebased this branch now